src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/alert-conditions.mdx

```diff
@@ -186,6 +188,7 @@ Creating a buffer:
   Key considerations:
    * <DoNotTranslate>**Popular use case**</DoNotTranslate>: A common use of gap filling is to insert a value of 0 for windows with no data received.
    * <DoNotTranslate>**Anomaly thresholds**</DoNotTranslate>: The gap-filling value is interpreted as the number of standard deviations from the last observed value when using anomaly thresholds. For example, filling gaps with a value of 0 would replicate the last value seen, effectively assuming no change.
+  * <DoNotTranslate>**Cost****</DoNotTranslate>: You can use a `WHERE` clause as a [gap-filling strategy](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions/#example-null-zero) but it's important to remember that these can increase data ingest, impacting your bill. 
```

Comment:
There's a lot hiding behind this bullet. I don't know if it can be solved in this PR, but if I were a customer thinking about using a where clause here, I'd want to know a LOT more.

Reply:
I made some changes to this.